utils/find_duplicates.py

```diff
@@ -139,34 +124,34 @@ def _normalize_content(self, content):
         return content
 
 
-
 class OvalDuplicatesFinder(DuplicatesFinder):
-
     def __init__(self, root_dir, specific_dirs_mask, shared_dir, shared_files_mask="*.xml"):
         DuplicatesFinder.__init__(self, root_dir, specific_dirs_mask, shared_dir, shared_files_mask)
 
-
     def _normalize_content(self, content):
         # remove comments
         # naive implementation (todo)
-        content = re.sub(r"^\s*#.*", "", content) # bash style comments - due to #platform
-        content = re.sub('<!--.*?-->', "", content, flags=re.DOTALL) # xml comments
+        content = re.sub(r"^\s*#.*", "", content)  # bash style comments - due to #platform
+        content = re.sub('<!--.*?-->', "", content, flags=re.DOTALL)  # xml comments
 
         # remove empty lines
         content = "\n".join([s for s in content.split("\n") if s])
 
         return content
 
 
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("root_ssg_directory", help="Path to root of ssg git repository")
+    args = parser.parse_args()
+    return args.root_ssg_directory
```

Comment:
In case there is only one argument that is being parsed I feel like we should still return `args` for consistency. It's much easier to extend the script if args is passed around. Plus I feel like it's easier to read the code - you will know that anything inside `args` is passed directly by the user.